app-backend/api/src/main/scala/exports/Routes.scala

```diff
@@ -78,10 +80,15 @@ trait ExportRoutes extends Authentication
   def createExport: Route = authenticate { user =>
     entity(as[Export.Create]) { newExport =>
       authorize(user.isInRootOrSameOrganizationAs(newExport)) {
-        onSuccess(write[Export](Exports.insertExport(newExport, user))) { export =>
-          kickoffProjectExport(export.id)
-          complete(export)
-        }
+        (for {
+          ei <- newExport.exportOptions.as[ExportOptions].toOption
+        } yield {
+          onSuccess(write[Export](Exports.insertExport(newExport, user))) { export =>
+            kickoffProjectExport(export.id)
+            complete(export)
+          }
+        }).getOrElse(
+          complete((StatusCodes.BadRequest, "Parameter 'exportOptions' could not be decoded to type 'ExportOptions'")))
```

Comment:
I don't think this is a very friendly error message. If this endpoint is user-facing at all, that user could be hitting the API with anything. Unless they're used to (de-)serialization concerns, this message is pretty opaque.